core/primitives/src/transaction.rs

```diff
@@ -233,6 +233,8 @@ impl Default for ExecutionStatus {
 struct PartialExecutionOutcome {
     pub receipt_ids: Vec<CryptoHash>,
     pub gas_burnt: Gas,
+    #[serde(with = "u128_dec_format")]
+    pub amount_burnt: Balance,
```

Comment:
How about using `tokens_burnt` or `near_burned`? "amount" term does not encode the unit of measurement. Having "gas_burnt" and "amount_burnt" can be confusing for the reader.

Reply:
renamed to `tokens_burnt`